misk-cron/src/main/kotlin/misk/cron/CronManager.kt

```diff
@@ -0,0 +1,136 @@
+package misk.cron
+
+import com.cronutils.model.CronType
+import com.cronutils.model.definition.CronDefinitionBuilder
+import com.cronutils.model.time.ExecutionTime
+import com.cronutils.parser.CronParser
+import com.squareup.moshi.Moshi
+import misk.jobqueue.JobQueue
+import misk.jobqueue.QueueName
+import misk.logging.getLogger
+import misk.moshi.adapter
+import java.time.Clock
+import java.time.Instant
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.util.PriorityQueue
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class CronManager @Inject constructor(moshi: Moshi) {
+  @Inject private lateinit var clock: Clock
+  @Inject private lateinit var jobQueue: JobQueue
+  @Inject @ForMiskCron private lateinit var queueName: QueueName
```

Comment:
For a service to use a cron in Misk, this impliues that they need to setup a job queue with the name that gets injected here?

Reply:
Correct - the CronJobHandler needs to be registered and the cron service registered plus individual crons registered as needed.